.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,83 @@
+---
+name: Release a new version
```

Comment:
In the Ignition repo we omit this header, which prevents the release checklist from showing up as an option in the new-bug flow.  We then put a [direct link](https://github.com/coreos/ignition/issues/new?labels=kind/release&template=release-checklist.md) in the developer docs.